lib/cbv2g/app_handshake/appHand_Decoder.c

```diff
@@ -26,8 +26,8 @@
 #include "cbv2g/common/exi_header.h"
 #include "cbv2g/common/exi_types_decoder.h"
 #include "cbv2g/app_handshake/appHand_Datatypes.h"
-
-
+#include "cbv2g/app_handshake/appHand_Decoder.h"
+#include "cbv2g/app_handshake/appHand_Encoder.h"
```

Comment:
Should be added in cbexigen
Here only `#include "cbv2g/app_handshake/appHand_Decoder.h"` is needed.